go/saltpack/packets.go

```diff
@@ -28,7 +28,7 @@ type encryptionHeader struct {
 	Type            MessageType    `codec:"type"`
 	Ephemeral       []byte         `codec:"ephemeral"`
 	SenderSecretbox []byte         `codec:"sendersecretbox"`
-	Receivers       []receiverKeys `codec:"rcvrs"`
+	Receivers       []receiverKeys `codec:"rcvrs,omitempty"`
```

Comment:
I don't think this `Receiver` list will/should ever be empty. That would mean that a message actually has no recipients -- there would be no PayloadKeyBoxes to open. It's the `receiverKeys.ReceiverKID` fields _inside_ the list that get emptied out in the anonymous case. We spec'd that the empty `ReceiverKID` should be a MessagePack `nil` value, so we might need to do some more plumbing to make that happen; I'll take a look.

I think I still prefer `nil` to an empty array, because it makes it less likely that a client could ignore the length, blindly pass the short array to NaCl, and start reading uninitialized memory. I've actually seen [_Python_ code](https://github.com/saltstack/libnacl/issues/58) do that.